golden_response/staging/get_stat_multi/latest.json

```diff
@@ -0,0 +1,102 @@
+{
+  "data": {
+    "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita": {
+      "stat": {
+        "country/FRA": {
+          "date": "2018",
+          "value": 46360,
+          "metadata": {
+            "importName": "WorldDevelopmentIndicators",
+            "provenanceUrl": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
+            "observationPeriod": "P1Y",
+            "unit": "InternationalDollar"
+          }
+        },
+        "country/USA": {
+          "date": "2018",
+          "value": 63690,
+          "metadata": {
+            "importName": "WorldDevelopmentIndicators",
+            "provenanceUrl": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
+            "observationPeriod": "P1Y",
+            "unit": "InternationalDollar"
+          }
+        },
+        "geoId/06": {},
+        "geoId/0649670": {}
+      }
+    },
+    "Count_CriminalActivities_CombinedCrime": {
+      "stat": {
+        "country/FRA": {},
+        "country/USA": {
+          "date": "2017",
+          "value": 8940603,
+          "metadata": {
+            "importName": "FBIGovCrime_AggCountry",
+            "provenanceUrl": "http://www.fbi.gov/services/cjis/ucr",
+            "observationPeriod": "P1Y"
+          }
+        },
+        "geoId/06": {
+          "date": "2017",
+          "value": 1164741,
+          "metadata": {
+            "importName": "FBIGovCrime",
+            "provenanceUrl": "http://www.fbi.gov/services/cjis/ucr",
+            "observationPeriod": "P1Y"
+          }
+        },
+        "geoId/0649670": {
+          "date": "2017",
+          "value": 2138,
+          "metadata": {
+            "importName": "FBIGovCrime",
+            "provenanceUrl": "http://www.fbi.gov/services/cjis/ucr",
+            "observationPeriod": "P1Y"
+          }
+        }
+      }
+    },
+    "Count_Person": {
+      "stat": {
+        "country/FRA": {
+          "date": "2020-06",
+          "value": 67081000,
+          "metadata": {
+            "importName": "WikipediaAndHumanCurated",
+            "provenanceUrl": "https://www.wikipedia.org",
+            "measurementMethod": "GoogleKGHumanCurated"
```

Comment:
We are ok exposing this?

Reply:
@pradh shall we do WikipediaAndHumanCurated for mmethod as well?